baseapp_chats/graphql/mutations.py

```
@@ -404,6 +388,213 @@ def mutate_and_get_payload(
+
+    @classmethod
+    def _resolve_rooms(
+        cls, info: graphene.ResolveInfo, room_ids: list[str]
+    ) -> tuple[list[Model], ErrorType | None]:
+        """Resolve relay ids to unique ChatRoom instances, preserving input order."""
+        rooms = []
+        seen_room_pks = set()
+        for room_id in room_ids:
+            try:
+                room = get_obj_from_relay_id(info, room_id)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)